manifests/install.pp

```diff
@@ -1,12 +1,16 @@
-# @summary Install pulpcore packages
-# Currently this is done with pip3. This should have the option to also install via RPMs.
+# @summary Install pulpcore packages, configure user and group, label ports
+#
 # @api private
 class pulpcore::install {
 
   package { 'python3-pulpcore':
     ensure => present,
   }
 
+  package { 'pulpcore-selinux':
```

Comment:
Our CI env in Travis doesn't have SELinux enabled because it's a container. You may run into this and in that case this (and the ports) probably need to be wrapped in `if $facts['os']['selinux']['enabled'] {}`

Reply:
To be clear: I prefer to use the types unconditionally. https://github.com/voxpupuli/puppet-selinux/commit/ff459f3747e496670a10485cfd2c22d3e96e97d9 is in puppet-selinux 3.1.0 so it should work without. I'd also expect the RPM to be installable.

Reply:
Great. I'll unrwap the ports and but leave the pulpcore-selinux package wrapped in that case.